diff --git a/backends/barcode/Zint.cpp b/backends/barcode/Zint.cpp
--- a/backends/barcode/Zint.cpp
+++ b/backends/barcode/Zint.cpp
@@ -67,1 +67,4 @@
-                symbol->input_mode = UNICODE_MODE;
+                symbol->input_mode =
+                        (symbology == BARCODE_CODE128)
+                                ? (UNICODE_MODE | EXTRA_ESCAPE_MODE)
+                                : UNICODE_MODE;
