diff --git a/glabels/File.hpp b/glabels/File.hpp
index 9e39a09..a1d2e35 100644
--- a/glabels/File.hpp
+++ b/glabels/File.hpp
@@ -16,6 +16,7 @@ namespace glabels
         public:
                 static bool newLabel( MainWindow *window = nullptr );
                 static void open( MainWindow *window );
+                static void openAsTemplate( MainWindow *window );
                 static void open( const QString& fileName, MainWindow *window );
                 static bool save( MainWindow *window );
                 static bool saveAs( MainWindow *window );
diff --git a/glabels/File.cpp b/glabels/File.cpp
index 0114b45..923dea8 100644
--- a/glabels/File.cpp
+++ b/glabels/File.cpp
@@ -143,6 +143,53 @@ namespace glabels
         }
 
 
+        ///
+        /// Open File As Template
+        ///
+        void File::openAsTemplate( MainWindow *window )
+        {
+                QString cwd = mCwd;
+                if ( window->model() && !window->model()->fileName().isEmpty() )
+                {
+                        QFileInfo fileInfo( window->model()->fileName() );
+                        if ( fileInfo.isFile() )
+                        {
+                                cwd = fileInfo.absolutePath();
+                        }
+                }
+
+                QString fileName =
+                        QFileDialog::getOpenFileName( window,
+                                                      tr("gLabels - Open As Template"),
+                                                      cwd,
+                                                      tr("glabels files (*.glabels);;All files (*)")
+                                );
+
+                if ( !fileName.isEmpty() )
+                {
+                        model::Model *model = model::XmlLabelParser::readFile( fileName );
+                        if ( model )
+                        {
+                                model->setFileName( QString() );
+                                model->setModified();
+
+                                if ( window->isEmpty() )
+                                {
+                                        window->setModel( model );
+                                }
+                                else
+                                {
+                                        auto *newWindow = new MainWindow();
+                                        newWindow->setModel( model );
+                                        newWindow->show();
+                                }
+
+                                mCwd = QFileInfo( fileName ).absolutePath();
+                        }
+                }
+        }
+
+
         ///
         /// Open file
         ///
diff --git a/glabels/MainWindow.hpp b/glabels/MainWindow.hpp
index 7ca2255..fa09c38 100644
--- a/glabels/MainWindow.hpp
+++ b/glabels/MainWindow.hpp
@@ -96,6 +96,7 @@ namespace glabels
 
                 void fileNew();
                 void fileOpen();
+                void fileOpenAsTemplate();
                 void fileOpenRecent();
                 void fileSave();
                 void fileSaveAs();
@@ -262,6 +263,7 @@ namespace glabels
 
                 QAction*  fileNewAction;
                 QAction*  fileOpenAction;
+                QAction*  fileOpenAsTemplateAction;
                 QAction*  fileSaveAction;
                 QAction*  fileSaveAsAction;
                 QAction*  fileShowEditorPageAction;
diff --git a/glabels/MainWindow.cpp b/glabels/MainWindow.cpp
index 07d8d84..e697767 100644
--- a/glabels/MainWindow.cpp
+++ b/glabels/MainWindow.cpp
@@ -13,6 +13,11 @@
                 fileOpenAction->setStatusTip( tr("Open an existing gLabels project") );
                 connect( fileOpenAction, SIGNAL(triggered()), this, SLOT(fileOpen()) );
 
+                fileOpenAsTemplateAction = new QAction( tr("Open as &Template..."), this );
+                fileOpenAsTemplateAction->setStatusTip(
+                        tr("Open an existing gLabels project as a new document") );
+                connect( fileOpenAsTemplateAction, SIGNAL(triggered()), this, SLOT(fileOpenAsTemplate()) );
+
                 for ( int i = 0; i < model::Settings::maxRecentFiles(); i++ )
                 {
                         auto* action = new QAction( this );
@@ -22,6 +27,7 @@
                 fileMenu = menuBar()->addMenu( tr("&File") );
                 fileMenu->addAction( fileNewAction );
                 fileMenu->addAction( fileOpenAction );
+                fileMenu->addAction( fileOpenAsTemplateAction );
                 fileRecentMenu = fileMenu->addMenu( QIcon::fromTheme( "glabels-file-recent" ), tr("Open Recent") );
                 for ( auto* action : fileRecentActionList )
                 {
@@ -33,6 +39,15 @@
         }
 
 
+        ///
+        /// File->Open As Template Action
+        ///
+        void MainWindow::fileOpenAsTemplate()
+        {
+                File::openAsTemplate( this );
+        }
+
+
         ///
         /// File->OpenRecent Action
         ///
