diff --git a/glabels/File.hpp b/glabels/File.hpp
--- a/glabels/File.hpp
+++ b/glabels/File.hpp
@@ -41,2 +41,3 @@
                 static void open( MainWindow *window );
+                static void openAsTemplate( MainWindow *window );
                 static void open( const QString& fileName, MainWindow *window );

diff --git a/glabels/File.cpp b/glabels/File.cpp
--- a/glabels/File.cpp
+++ b/glabels/File.cpp
@@ -143,0 +144,46 @@
+
+        ///
+        /// Open File As Template
+        ///
+        void File::openAsTemplate( MainWindow *window )
+        {
+                QString cwd = mCwd;
+                if ( window->model() && !window->model()->fileName().isEmpty() )
+                {
+                        QFileInfo fileInfo( window->model()->fileName() );
+                        if ( fileInfo.isFile() )
+                        {
+                                cwd = fileInfo.absolutePath();
+                        }
+                }
+
+                QString fileName =
+                        QFileDialog::getOpenFileName( window,
+                                                      tr("gLabels - Open As Template"),
+                                                      cwd,
+                                                      tr("glabels files (*.glabels);;All files (*)")
+                                );
+
+                if ( !fileName.isEmpty() )
+                {
+                        model::Model *model = model::XmlLabelParser::readFile( fileName );
+                        if ( model )
+                        {
+                                model->setFileName( QString() );
+                                model->setModified();
+
+                                if ( window->isEmpty() )
+                                {
+                                        window->setModel( model );
+                                }
+                                else
+                                {
+                                        auto *newWindow = new MainWindow();
+                                        newWindow->setModel( model );
+                                        newWindow->show();
+                                }
+
+                                mCwd = QFileInfo( fileName ).absolutePath();
+                        }
+                }
+        }

diff --git a/glabels/MainWindow.hpp b/glabels/MainWindow.hpp
--- a/glabels/MainWindow.hpp
+++ b/glabels/MainWindow.hpp
@@ -95,2 +95,3 @@
         void fileOpen();
+        void fileOpenAsTemplate();
         void fileOpenRecent();
@@ -261,2 +262,3 @@
         QAction*  fileOpenAction;
+        QAction*  fileOpenAsTemplateAction;
         QAction*  fileSaveAction;

diff --git a/glabels/MainWindow.cpp b/glabels/MainWindow.cpp
--- a/glabels/MainWindow.cpp
+++ b/glabels/MainWindow.cpp
@@ -285,9 +285,14 @@
                 fileOpenAction = new QAction( tr("&Open..."), this );
                 fileOpenAction->setIcon( QIcon::fromTheme( "glabels-file-open" ) );
                 fileOpenAction->setShortcut( QKeySequence::Open );
                 fileOpenAction->setStatusTip( tr("Open an existing gLabels project") );
                 connect( fileOpenAction, SIGNAL(triggered()), this, SLOT(fileOpen()) );

+                fileOpenAsTemplateAction = new QAction( tr("Open as &Template..."), this );
+                fileOpenAsTemplateAction->setStatusTip(
+                        tr("Open an existing gLabels project as a new document") );
+                connect( fileOpenAsTemplateAction, SIGNAL(triggered()), this, SLOT(fileOpenAsTemplate()) );
+
                 for ( int i = 0; i < model::Settings::maxRecentFiles(); i++ )
                 {
                         auto* action = new QAction( this );
@@ -611,7 +616,9 @@
                 fileMenu = menuBar()->addMenu( tr("&File") );
                 fileMenu->addAction( fileNewAction );
                 fileMenu->addAction( fileOpenAction );
+                fileMenu->addAction( fileOpenAsTemplateAction );
                 fileRecentMenu = fileMenu->addMenu( QIcon::fromTheme( "glabels-file-recent" ), tr("Open Recent") );
                 for ( auto* action : fileRecentActionList )
                 {
                         fileRecentMenu->addAction( action );
@@ -1177,11 +1183,19 @@
         /// File->Open Action
         ///
         void MainWindow::fileOpen()
         {
                 File::open( this );
         }

+        ///
+        /// File->Open As Template Action
+        ///
+        void MainWindow::fileOpenAsTemplate()
+        {
+                File::openAsTemplate( this );
+        }
+
 
         ///
         /// File->OpenRecent Action
         ///
