diff --git a/glabels/File.hpp b/glabels/File.hpp
--- a/glabels/File.hpp
+++ b/glabels/File.hpp
@@ -41,2 +41,3 @@
                 static void open( MainWindow *window );
+                static void openAsTemplate( MainWindow *window );
                 static void open( const QString& fileName, MainWindow *window );

diff --git a/glabels/File.cpp b/glabels/File.cpp
--- a/glabels/File.cpp
+++ b/glabels/File.cpp
@@ -132,0 +133,46 @@
+
+        ///
+        /// Open File As Template
+        ///
+        void File::openAsTemplate( MainWindow *window )
+        {
+                QString cwd = mCwd;
+                if ( window->model() && !window->model()->fileName().isEmpty() )
+                {
+                        QFileInfo fileInfo( window->model()->fileName() );
+                        if ( fileInfo.isFile() )
+                        {
+                                cwd = fileInfo.absolutePath();
+                        }
+                }
+
+                QString fileName =
+                        QFileDialog::getOpenFileName( window,
+                                                      tr("gLabels - Open As Template"),
+                                                      cwd,
+                                                      tr("glabels files (*.glabels);;All files (*)")
+                                );
+
+                if ( !fileName.isEmpty() )
+                {
+                        model::Model *model = model::XmlLabelParser::readFile( fileName );
+                        if ( model )
+                        {
+                                model->setFileName( QString() );
+                                model->setModified();
+
+                                if ( window->isEmpty() )
+                                {
+                                        window->setModel( model );
+                                }
+                                else
+                                {
+                                        auto *newWindow = new MainWindow();
+                                        newWindow->setModel( model );
+                                        newWindow->show();
+                                }
+
+                                mCwd = QFileInfo( fileName ).absolutePath();
+                        }
+                }
+        }
+

diff --git a/glabels/MainWindow.hpp b/glabels/MainWindow.hpp
--- a/glabels/MainWindow.hpp
+++ b/glabels/MainWindow.hpp
@@ -95,2 +95,3 @@
         void fileOpen();
+        void fileOpenAsTemplate();
         void fileOpenRecent();
@@ -261,2 +262,3 @@
         QAction*  fileOpenAction;
+        QAction*  fileOpenAsTemplateAction;
         QAction*  fileSaveAction;

diff --git a/glabels/MainWindow.cpp b/glabels/MainWindow.cpp
--- a/glabels/MainWindow.cpp
+++ b/glabels/MainWindow.cpp
@@ -263,0 +264,5 @@
+                fileOpenAsTemplateAction = new QAction( tr("Open as &Template..."), this );
+                fileOpenAsTemplateAction->setStatusTip(
+                        tr("Open an existing gLabels project as a new document") );
+                connect( fileOpenAsTemplateAction, SIGNAL(triggered()), this, SLOT(fileOpenAsTemplate()) );
+
@@ -612,0 +618 @@
+                fileMenu->addAction( fileOpenAsTemplateAction );
@@ -1173,0 +1180,9 @@
+
+        ///
+        /// File->Open As Template Action
+        ///
+        void MainWindow::fileOpenAsTemplate()
+        {
+                File::openAsTemplate( this );
+        }
+
